ui/src/main/js/application/MainPage.js

```diff
@@ -246,7 +251,14 @@ const MainPage = ({
                 urlName={CERTIFICATE_INFO.url}
                 descriptor={globalDescriptorMap[CERTIFICATE_INFO.key]}
                 render={() => (
-                    <CertificatesPage />
+                    <BetaPage betaSelected>
+                        <BetaComponent>
+                            <CertificatesPageLayout />
+                        </BetaComponent>
+                        <CurrentComponent>
+                            <CertificatesPage />
+                        </CurrentComponent>
+                    </BetaPage>
```

Comment:
Is there additional work planned for the certificates page? If not, I think we can probably drop the beta page if we're just converting to the new table format. Otherwise, we may just want a ticket as a reminder to clear up the BetaPage before release so we don't forget

Reply:
For testing purposes, I'll leave the BetaPage setup in.  I am making note that this will be removed before merging into master.